qcodes/dataset/sqlite/queries.py

```diff
@@ -1593,7 +1593,10 @@ def update_GUIDs(conn: ConnectionPlus) -> None:
 
     log.info('Commencing update of all GUIDs in database')
 
-    cfg = Config()
+    cfg = qc.config
+
+    if cfg is None:
```

Comment:
I don't think this is needed. `qc.config.current_config` can be None since it has the type `Optional[DotDict]` but qc.config is always a `Config` object

Reply:
Yes I forgot to remove that. I will push once the local run of tests finish.